opensearchpy/_async/http_aiohttp.py

```diff
@@ -246,12 +246,8 @@ async def perform_request(
         else:
             query_string = ""
 
-        # There is a bug in aiohttp that disables the re-use
-        # of the connection in the pool when method=HEAD.
-        # See: aio-libs/aiohttp#1769
         is_head = False
```

Comment:
I believe this is no longer needed as we only reuse it once, maybe just check the verb where it's needed?